composer.json

```diff
@@ -29,8 +30,7 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.4",
-        "slevomat/coding-standard": "^6.1",
-        "squizlabs/php_codesniffer": "^3.3"
+        "slevomat/coding-standard": "dev-master#47cf967b284f7ff41ab500ae55cdbcee986b1534"
```

Comment:
There is no real tag yet. But I created a follow up ticket for fixing this in 4.1.0

Reply:
OK imho as long we use a referenced commit.